src/helpers/ides.js

```diff
@@ -154,6 +154,27 @@ module.exports = {
     ]).then(v => utils.determineFound('VSCode', v[0], v[1]));
   },
 
+  getVisualStudioInfo: () => {
+    if (utils.isWindows) {
+      return utils
+        .run(
+          `"${process.env['ProgramFiles(x86)']}/Microsoft Visual Studio/Installer/vswhere.exe" -format json -prerelease`
+        )
+        .then(jsonText =>
+          JSON.parse(jsonText).map(_ => {
```

Comment:
can you give _ a more meaningful name? Even if it is just an initial of another variable name.